ci/run_tests.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 The CapableVMs "CHERI Examples" Contributors.
+# SPDX-License-Identifier: MIT OR Apache-2.0
+#
+# Test that examples execute as expected.
+#
+# This is designed to be run by .buildbot.sh, but can also be run manually too,
+# as long as you have a suitable environment, and an execution platform set up
+# and running so that the make-based run-* targets have something to connect to.
+# Notable environment variables:
+#
+# - BUILDBOT_PLATFORM
+#   - Used to find the example makefile. Set to something like "morello-purecap"
+#     or "riscv64-purecap".
+# - SSHPORT
+# - SSHUSER (defaults to 'root')
+# - SSHHOST (defaults to 'localhost')
+#
+# TODO: Extend this to cover interactive examples and other architectures. 
+# For now, we just test examples on `morello-hybrid`.
+
+set -e
+
+ERROR=false
+if [ -z "$SSHPORT" ]; then
+    echo "SSHPORT is empty or not set" >&2
+    ERROR=true
+fi
+if [ -z "$BUILDBOT_PLATFORM" ]; then
+    echo "BUILDBOT_PLATFORM is empty or not set" >&2
+    ERROR=true
+fi
+$ERROR && exit 1
+
+#One time setup per architecture
```

Comment:
Space here before the capital letter.

Reply:
Done in [2cc9b31d](https://github.com/capablevms/bdwgc/pull/11/commits/2cc9b31d999e78ceced82eadae95bcd57306d72a)